packages/expo-cli/src/commands/build/ios/IOSBuilder.ts

```diff
@@ -41,6 +41,8 @@ class IOSBuilder extends BaseBuilder {
 
   async run(): Promise<void> {
     await this.validateProject();
+    this.maybeWarnDamagedSimulator();
```

Comment:
We can push this below the build in progress check too. The reason here is to display it ASAP when it makes sense (after validation).

Reply:
can we check operating system version to only show this on catalina+?

Reply:
Done 😄

Reply:
can you also create an entry on expo.fyi that explains this and maybe provides some context on why it happens?